src/App.js

```diff
@@ -46,7 +48,21 @@ class App extends Component {
 				alert('Something went wrong, navigate back to the homepage')
 			})
 		}
-		return <MovieShowPage movie={this.state.foundMovie} />
+		return <MovieShowPage movie={this.state.foundMovie} userMovieRating={this.state.ratings} userID={this.state.id} getUserRatings={this.getUserRatings} />
+	}
+
+	getUserRatings = () => {
```

Comment:
When we get into async testing tomorrow, think about what it would look like to break some of this function into another file.

Reply:
Noted!